### ISSUE TITLE
"Modify jQuery" section is incorrect

### ISSUE THREAD

**Issue by ihorvorotnov** [taniarascia/wp-functions#31]
The ["Modify jQuery"](https://github.com/taniarascia/wp-functions#modify-jquery) function is doing it wrong. See ["Notes" section](https://developer.wordpress.org/reference/functions/wp_register_script/#notes) for the wp_register_script() function documentation:

> The function should be called using the wp_enqueue_scripts action hook if you want to call it on the front-end of the site. To call it on the administration screens, use the admin_enqueue_scripts action hook. For the login screen, use the login_enqueue_scripts action hook. Calling it outside of an action hook can often lead to unexpected results and should be avoided.

You can get some additional insight from the old blog post by Pippin [here as well](https://pippinsplugins.com/loading-scripts-correctly-in-the-wordpress-admin/).

In short, registering, and even more - enqueueing scripts on `init` hook is wrong. I'll submit a PR covering all cases when I get a minute. Also, it may conflict with #30

**Comment by viktorbijlenga** [comment]
What a coincidence. I submitted a Pull Request for this today. https://github.com/taniarascia/wp-functions/pull/30

**Comment by ihorvorotnov** [comment]
@viktorbijlenga Yes, I've got notification about your PR, that's why I actually had a look into the code and chimed in. Your PR replaces the `init` hook (which is correct) but does not mention non-frontend cases - admin and login, and doesn't explain why the change is being made. 

Having seen your PR is why I said if I'm submitting a PR covering all scenarios, it might conflict with yours. Actually, you can amend your commit with `admin_enqueue_scripts` and `login_enqueue_scripts` plus some inline docs explaining the cases and close this issue.

**Comment by viktorbijlenga** [comment]
I read your comment a bit to fast, so saw your mention a bit to late.

Feel free to improve or refactor my code with a new pull request if you find the time. Your additions sounds like great improvements to me.

**Comment by ihorvorotnov** [comment]
No worries, I'll do it in the next few days.
